src/commands/dockersupport/localDockerSupport.ts

```diff
@@ -14,51 +14,43 @@ import { validateDockerInstalled } from "./validateDockerInstalled";
  * @param node - Function App Project
  */
 export async function localDockerPrompt(context: IActionContext, devContainerFolderPathUri: vscode.Uri, node?: SlotTreeItemBase, devContainerName?: string): Promise<void> {
-    if (node) {
-        // check if language is supported for dev container
-        if (devContainerName) {
-            // check if the function app is in Linux
-            if (node.site.reserved) {
-                // asks if the user wants to use Docker for initializing the project locally
-                const useDocker: string = await prompt(context);
-                if (useDocker === "yes") {
-                    // download dev container
-                    await requestUtils.downloadFile(
-                        `https://raw.githubusercontent.com/microsoft/vscode-dev-containers/master/containers/${devContainerName}/.devcontainer/devcontainer.json`,
-                        vscode.Uri.joinPath(devContainerFolderPathUri, 'devcontainer.json').fsPath
-                    );
-                    // download docker file
-                    await requestUtils.downloadFile(
-                        `https://raw.githubusercontent.com/microsoft/vscode-dev-containers/master/containers/${devContainerName}/.devcontainer/Dockerfile`,
-                        vscode.Uri.joinPath(devContainerFolderPathUri, 'Dockerfile').fsPath
-                    );
-                    // external - check if Docker is installed, Remote Development extension AND Docker Extension
-                    if (!validateDockerInstalled()) {
-                        // if Docker is not downloaded - ask the user if they'd like to download Docker
-                        const downloadDocker: string = await prompt2(context);
-                        if (downloadDocker === "yes") {
-                            // Check if Operating System is Windows
-                            if (process.platform == "win32") {
-                                // Download Docker with an MSI package
-                            } else {
-                                // Not windows: display link to download docker externally from Docker documentation
-                                void vscode.window.showInformationMessage(localize('downloadDocker', 'Check the (Docker documentation)[https://docs.docker.com/get-docker/] to download Docker for your system'));
-                            }
-                        } else {
-                            void vscode.window.showInformationMessage(localize('noDocker', 'Continuing without the use of Docker as user requested'));
-                        }
+    if (node && devContainerName && node.site.reserved) {
+        // asks if the user wants to use Docker for initializing the project locally
+        const useDocker: string = await prompt(context);
+        if (useDocker === "yes") {
```

Comment:
we should make this check case insensitive. as in something like `userDocker.ToLower() == "yes"`.  Is the 'yes'/'no' typed manually by the user or is it  a dropdown like selection?

Reply:
It is a dropdown selection. Is it still necessary to add this check?

Reply:
incase of dropdown, no need to worry about case check.